utils/Sounds.ts

```diff
@@ -0,0 +1,49 @@
+import { Audio } from "expo-av";
+
+enum Sounds {
+  NEXT = require("app/assets/sounds/nextbeep.mp3"),
+  FIRST = require("app/assets/sounds/firstbeep.mp3"),
+}
+
+const firstSound = new Audio.Sound();
+const nextSound = new Audio.Sound();
+
+async function loadSound() {
+  // Loading Sound
+  try {
+    await firstSound.loadAsync(Sounds.FIRST);
+    await nextSound.loadAsync(Sounds.NEXT);
+  } catch (error) {
+    console.log(error);
+  }
+}
```

Comment:
We might wanna set console or something and check if this function only gets called once. Otherwise, we might wanna do this logic within UI component. Don't worry about this if you are sure it doesn't get called multiple times!

Reply:
I tried to create the `Audio` objects when you press the count button. If I can succeed, there would not be a need for a `loadSound` function.